pommerman/network/ion_server/match.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+"""IonServer Match handler - This contains functions responsible for playing matches (You shouldn't use this file directly)"""
+
+# Imports
+from multiprocessing import Pipe
+from .constants import SPCommands as spc
+from uuid import uuid4
+from os import listdir, makedirs
+from numpy import array
+from rapidjson import dumps, dump
+from gzip import compress
+from enum import Enum
+from numpy import ndarray, uint8, int64
+import pommerman
+
+
+# Functions
+def unique_uuid(dir):
+    """Generates a unique UUID and checks for collision"""
```

Comment:
Add period.
Also, Args.

Reply:
^^

Reply:
Wait, so you did neither and just resolved it? -_-

Reply:
I replied to this somewhere else that's when I use "^^". The reason was that this file shouldn't be used directly: https://github.com/MultiAgentLearning/playground/blob/feb916625595bd61c9ccb621a471d98cf94c0ea6/pommerman/network/ion_server/match.py#L5. So, there's not exactly much point in elaborating upon the arguments. About the period: I'll have to change every single comment to have a period then and that's a fair amount of work for nothing all that important.

Reply:
Sigh, that's really not how this works. You do the effort now to make it as legible as possible, so that later everything is much easier to understand. My qualm here is not really with the period - I added that on when I was suggesting the other change. 
My qualm is that I have no idea what directories `dir` is supposed to be. It's this random input that was passed and has seemingly no requirements on it. What should I be giving this function?

Reply:
But also do you even need to do that? How likely is it that two uuid.uuid4s collide given that you're probably not making a very large amt of them?

Reply:
This function is more or less general. It checks any directory you give it to check if UUIDs collide. It's used for getting match IDs to check the match directory if you go to the main logic of the file.
```But also do you even need to do that? How likely is it that two uuid.uuid4s collide given that you're probably not making a very large amt of them?```
I'd rather not overwrite a pre-existing file by mistake. The function doesn't use up any substantial amount of resources so I doubt it's going to make a difference by removing it.

Reply:
Say something like that in the doc blocks. I totally get what is happening with that last comment.